src/classes/UTIL_Query_TEST.cls

```diff
@@ -350,10 +350,36 @@ private class UTIL_Query_TEST {
             .withSelectFields(selectFields)
             .withFrom('Contact')
             .build();
-    
+
         String expectedSoql = 'SELECT Id, AccountId, SaLuTaTiOn, FirstName, LastName FROM Contact';
 
         System.assertEquals(expectedSoql, soql, 'SOQL should not include toLabel for Salutation field');
     }
-	
+
+    /*********************************************************************************************************
+    @description
+        Tests SOQL builder for whenMultiCurrencyField()
+    verify:
+        The SOQL contains CurrencyISOCode if in a Multi Currency Org
+    **********************************************************************************************************/
+    @isTest
+    private static void testSoqlIncludesMultiCurrencyOrgIfMultiCurrencyEnabled() {
+        String[] selectFields = new String[]{
+            'Id', 'Amount'
+        };
+
+        String soql = new UTIL_Query()
+            .withSelectFields(selectFields)
+            .withMultiCurrencyField()
+            .withFrom('Opportunity')
+            .build();
+
+        if (UserInfo.isMultiCurrencyOrganization()) {
+            System.assert(soql.contains('CurrencyIsoCode'), 'Multi Currency Org: CurrencyIsoCode should be present. SOQL: ' + soql);
```

Comment:
Probably best to use the constant since it's defined and being used for the method